tests/project/models.py

```diff
@@ -30,16 +44,44 @@ def version(self) -> str:
         return str(self.pk) + ":" + str(self.date_modified)
 
 
+if TYPE_CHECKING:
+
+    class ChildQuerySet(ExtendedQuerySet["Child"]):
+        ...
+
+else:
+
+    class ChildQuerySet(ExtendedQuerySet):
+        ...
+
+
+ChildManager = BaseManager.from_queryset(ChildQuerySet)
```

Comment:
A new manager could still be built from the `Manager` type, right?

```suggestion
ChildManager = Manager.from_queryset(ChildQuerySet)
```